schemas/data/conversions.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/advertising/conversions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "conversions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metric"
+  ],
+  "description": "The advertising metric conversions indicates a predefined conversion event is triggered by a surfer online",
```

Comment:
Is a conversion not analysis specific. The observed events may show that the ad was consumed to 75%. Maybe today that is considered a conversion, but tomorrow it is considered a conversion only when it hits 90%.

Reply:
Yes, you are right in a way, however this metric is describing/counting the trigger event itself that carries the metadata leading to a Conversion. There are post-event processes which look over all these Experiences Events and either attribute or not this event to a real world Conversion stitching together different events in a path to conversion.

Reply:
Ok, got it.